superset/databases/api.py

```diff
@@ -1204,3 +1210,58 @@ def validate_parameters(self) -> FlaskResponse:
         command = ValidateDatabaseParametersCommand(payload)
         command.run()
         return self.response(200, message="OK")
+
+    @expose("/<int:pk>/ssh_tunnel/", methods=["DELETE"])
+    @protect()
+    @safe
```

Comment:
IIRC, we no longer want to use this decorator because we want all errors to raise to be SIP41 compliant. DId I get that right @betodealmeida?

Reply:
+1 we should remove this so SIP-41 errors can be returned